src/AWS/Signers/V4.elm

```diff
@@ -52,7 +54,7 @@ formatDate date =
     date
         |> formatISO8601
         |> Regex.replace All
-            (regex "([-:\\]|\\.\\d{3})")
+            (regex "[-:]")
```

Comment:
I guess the `.ddd` is not necessary if our dates will never have the micro-seconds. This is worth unit testing